client-ts/src/organisms/mypage/marketer/shared/UrlUploadDialog.tsx

```diff
@@ -91,7 +91,7 @@ export default function UrlUploadDialog(props: UrlUploadDialogProps): JSX.Elemen
       open={open}
       maxWidth="sm"
       fullWidth
-      title="URL 등록"
+      title="랜딩페이지 URL 등록"
```

Comment:
요고 광고페이지 라고 해줘야하지 않을까요?
```suggestion
      title="광고페이지 URL 등록"
```

Reply:
아닙니다 광고주가 캠페인 등록할 때, 광고주의 랜딩페이지 URL을 받는 것이기 때문에 랜딩페이지가 맞습니다